composer.json

```diff
@@ -14,24 +14,24 @@
     ],
 
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.0",
         "ext-mbstring": "*",
         "behat/gherkin": "^4.10.0",
-        "behat/transliterator": "^1.2",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/event-dispatcher": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/translation": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/yaml": "^4.4 || ^5.0 || ^6.0 || ^7.0",
+        "behat/transliterator": "^1.5",
+        "symfony/console": "^5.0 || ^6.0 || ^7.0",
+        "symfony/config": "^5.0 || ^6.0 || ^7.0",
+        "symfony/dependency-injection": "^5.0 || ^6.0 || ^7.0",
+        "symfony/event-dispatcher": "^5.0 || ^6.0 || ^7.0",
+        "symfony/translation": "^5.0 || ^6.0 || ^7.0",
+        "symfony/yaml": "^5.0 || ^6.0 || ^7.0",
```

Comment:
symfony versions should all be `^5.4 || ^6.4 || ^7.0` - same in the dev dependencies